layers/meta-balena-compulab/recipes-core/images/resin-image.inc

```diff
@@ -29,4 +29,3 @@ IMAGE_INSTALL_append_cl-som-imx8 = " \
 IMAGE_CMD_resinos-img_append_cl-som-imx8 () {
     dd if=${DEPLOY_DIR_IMAGE}/imx-boot-${MACHINE}-sd.bin-flash_evk of=${RESIN_RAW_IMG} conv=notrunc seek=33 bs=1K
 }
-
```

Comment:
Let's leave a trailing empty line like we do in all our recipes :)

Reply:
done